lib/ansiblelint/formatters/__init__.py

```diff
@@ -2,6 +2,7 @@
     from ansible import color
 except ImportError:
     from ansible.utils import color
+from ansiblelint.utils import normpath
```

Comment:
Let's add an extra line to separate local imports from third party
```suggestion

from ansiblelint.utils import normpath
```